src/constants/categories.js

```diff
@@ -11,21 +11,57 @@ export default [
         name: 'Employment',
         tint: '040',
         shade: '600',
-        indicatorIds: [
-          'participationRate',
-          'gdp',
+        summaryIndicatorIds: [
+          'unemploymentRateGeneral',
+          'unemploymentRateIndigenous',
+          'employmentGrowth',
+        ],
+        charts: [
+          {
+            name: 'Youth unemployment',
+            indicatorIds: [
+              'unemploymentRateYouth',
+            ],
+          },
+          {
+            name: 'Unemployment rate',
+            indicatorIds: [
+              'unemploymentRateGeneral',
+            ],
+          },
+          {
+            name: 'Indigenous unemployment',
+            indicatorIds: [
+              'unemploymentRateIndigenous',
+            ],
+          },
+          {
+            name: 'Participation rate',
+            indicatorIds: [
+              'participationRateGeneral',
+              'participationRateFemale',
+              'participationRateMale',
+            ],
```

Comment:
This needs a mechanism to feed through the text that goes in the tooltip (`longDescription`). If you do this here:
```js
{
  name: 'Participation rate',
  longDescription: 'This is the text that will show in the tooltip for this chart',
  indicatorIds: [
    'participationRateGeneral',
    'participationRateFemale',
    'participationRateMale',
  ],
},
```

And then in `<SubCategoryDetails>` and wherever else you render the chart you do this:
```jsx
<CityColumnChart
  title={chart.name}
  longDescription={chart.longDescription}
  cities={props.cities}
  colorBase={props.colorName}
  colorVariation={sc.shade}
  indicatorIds={chart.indicatorIds}
/>
```